instrumentation/kafka-clients/src/main/java/brave/kafka/clients/TracingConsumer.java

```diff
@@ -20,12 +20,10 @@
 import brave.propagation.TraceContext.Injector;
 import brave.propagation.TraceContextOrSamplingFlags;
 import brave.sampler.SamplerFunction;
+
+import java.lang.reflect.Method;
 import java.time.Duration;
-import java.util.Collection;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
+import java.util.*;
```

Comment:
Could we go back to explicit imports please? This is the pattern we follow in this repo.

Reply:
Done